tests/bluetooth/host/buf/mocks/buf_help_utils.c

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/zephyr.h>
+#include <zephyr/bluetooth/buf.h>
+#include "net_buf.h"
+#include "buf_help_utils.h"
+
+void validate_net_buf_alloc_called_behaviour(struct net_buf_pool *pool, k_timeout_t *timeout)
```

Comment:
I would rename that one: `void expect_single_call_net_buf_alloc(...)` (and the others using the same style)